osgtest/tests/test_58_gfal2util.py

```diff
@@ -58,3 +69,15 @@ def test_03_remove_server_file_gfal2_util(self):
         self.assert_(file_removed, 'Copied file still exists')
         files.remove(TestGFAL2Util.__temp_dir)
 
+    def test_04_copy_local_to_server_gfal2_gftp_util(self):
+        core.skip_ok_unless_installed('osg-gridftp', 'gfal2-plugin-gridftp')
```

Comment:
@efajardo could you submit a new PR where this test depends on {{globus-gridftp-server-progs}} instead of {{osg-gridftp}}?